curriculum/Chapter-?-Databases.md

```diff
@@ -0,0 +1,30 @@
+# Chapter ?: Databases
+
+## Objectives
+
+After completing this chapter, you will be able to:
+
+- Understand the role and importance of databases in application development.
```

Comment:
Possibly remove this "Understand the role and importance of databases in application development"